<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImadreamClass</class>
 <widget class="QMainWindow" name="ImadreamClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1307</width>
    <height>743</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Imadream</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>120</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>工具栏</string>
    </property>
    <widget class="QPushButton" name="pushButton_pen">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>画笔</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_eraser">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>80</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>橡皮</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_palette">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>31</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 0, 0);</string>
     </property>
     <property name="text">
      <string>color</string>
     </property>
    </widget>
    <widget class="QSlider" name="Slider_pensize">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>91</width>
       <height>22</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_image">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>1101</width>
      <height>661</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>false</bool>
    </property>
    <widget class="Ximage" name="imageWindow">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1099</width>
       <height>659</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1307</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="text">
    <string>打开</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="action_save">
   <property name="text">
    <string>保存</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Ximage</class>
   <extends>QWidget</extends>
   <header>ximage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="imadream.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_palette</sender>
   <signal>clicked()</signal>
   <receiver>ImadreamClass</receiver>
   <slot>setColor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>54</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>775</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pen</sender>
   <signal>clicked()</signal>
   <receiver>ImadreamClass</receiver>
   <slot>setPen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>99</x>
     <y>755</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_eraser</sender>
   <signal>clicked()</signal>
   <receiver>ImadreamClass</receiver>
   <slot>setEraser()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>-43</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Slider_pensize</sender>
   <signal>valueChanged(int)</signal>
   <receiver>ImadreamClass</receiver>
   <slot>setBrushSize()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>-3</x>
     <y>473</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setPen()</slot>
  <slot>setColor()</slot>
  <slot>setEraser()</slot>
  <slot>setBrushSize()</slot>
 </slots>
</ui>
